exercises/practice/series/.meta/Example.roc

```diff
@@ -0,0 +1,25 @@
+module [slices]
+
+slices : Str, U64 -> Result (List Str) [StringArg [WasEmpty, WasNotAscii Str], SliceLengthArg [WasZero, WasTooLarge U64 Str]]
+slices = \string, sliceLength ->
+    chars = string |> Str.toUtf8
+    len = chars |> List.len
+    if len == 0 then
+        StringArg WasEmpty |> Err
+    else if sliceLength == 0 then
+        SliceLengthArg WasZero |> Err
+    else if sliceLength > len then
+        SliceLengthArg (WasTooLarge sliceLength string) |> Err
+    else
```

Comment:
I like the idea of all of these cases just returning an empty list as it seems like a reasonable answer each time. For example, when I ask 'How many length 5 substrings are there in ""', none seems like a good answer.

Reply:
Yes, I agree for these 3 cases. Regarding the UTF8 error, that's a bit trickier:
* Should we just crash with a message saying that only ASCII strings are supported? (that's what I just did)
* Should we use the unicode package to split into graphemes? It feels like this would be cleaner. We would not have to force the users to use the unicode pacakge, and we would not add UTF8 tests, but at least if users want to go the extra mile and support unicode strings, then can.
* Or should we just return an `Err (BadUtf8 _ _)`?

Wdyt?

Reply:
Mmmh... actually `Grapheme.split` can also fail, so we would still need to crash or return an error, it doesn't help much. I think I'll just leave it as is (i.e., crash if non-ASCII), wdyt?

Reply:
I think crashing on non ascii is the right thing to do for these Exercises. It keeps things simple that way.